packages/frontend-2/components/viewer/anchored-point/thread/CommentAttachments.vue

```diff
@@ -7,7 +7,9 @@
       size="sm"
       @click="() => onAttachmentClick(attachment)"
     >
-      <span class="truncate relative bottom-0.5">{{ attachment.fileName }}</span>
+      <span class="truncate relative bottom-0.5 text-xs">
```

Comment:
better to use our semantic font sizes instead of text-xs directly

Reply:
Semantic Font Sizes - To be returned to